dissect/target/plugins/os/windows/regf/mru.py

```diff
@@ -419,3 +461,13 @@ def parse_office_mru_key(target, key, record):
             _user=user,
             _key=key,
         )
+
+
+def get_filepath_from_bag(parsed_bag):
```

Comment:
Can you add type hints? At least to the new code, but we would really appreciate it if you added it to the entire file <3.

Reply:
Rebasing from `main` will take care of this.